weasyl/searchtag.py

```diff
@@ -17,6 +17,15 @@
 
 _TAG_DELIMITER = re.compile(r"[\s,]+")
 
+_SEARCHTAG_BLACKLIST_REGEXP_PATTERN = re.compile(r"""
```

Comment:
Rejecting on `tag.count("*") > 1 or "*" in tag and len(tag) < 3` or something might be worth considering.